solutions/devicesimulation-nohub/single-vm/setup.sh

```diff
@@ -1,243 +1,15 @@
 #!/usr/bin/env bash
 # Copyright (c) Microsoft. All rights reserved.
 # Note: Windows Bash doesn't support shebang extra params
-set -ex
 
-APP_PATH="/app"
```

Comment:
Instead of deleting section and adding to new file. Use `git mv setup.sh setupInternal.sh` command to rename the file to setupInternal.sh. Then make changes to setup.sh

Reply:
thanks, updated.